bitnami/ghost/values.yaml

```diff
@@ -601,6 +602,7 @@ mariadb:
     username: bn_ghost
     password: ""
     existingSecret: ""
+    existingSecretPasswordKey: "mariadb-password"
```

Comment:
Values under the `mariadb` section are values that exist in the MariaDB chart. We should add this value to the MariaDB chart before adding it to Ghost.

Reply:
@alemorcuq Good catch. PR for mariadb chart is up #8602